dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/FileAsset.java

```diff
@@ -132,16 +132,16 @@ private Dimension computeFileDimension(final File file) {
 		if (fileDimension == null) {
 			try {
     				return (fileDimension = ImageUtil.getInstance().getDimension(file));
-			} catch (Exception e) {
-				Logger.debug(this,
+			} catch (Throwable e) {
```

Comment:
Codacy found an issue: [A catch statement should never catch throwable since it includes errors.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5918493)